### ISSUE TITLE
font-weight numeric does not catch typos

### ISSUE THREAD

**Issue by paroxyzm** [stylelint/stylelint#6591]
### What steps are needed to reproduce the bug?

More of a question than a bug. Maybe it is behaving as intended.

[A link to a working example](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdilAzCmALgLQDuMAlgOYAWh8KARgDZQDcWAvliADQgBnAJ5EkADwTgBAvuAK5qkjNhToQAJwCuzGALUNlOHGvxEylWiUwRCSQhQL7VITJoC2MdRTBqsRlNyYnCCcQA)

CSS source:
`
a {
  font-weight: bld;
}
`

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "font-weight-notation": "numeric"
    }
}

```

### How did you run Stylelint?

CLI

### Which version of Stylelint are you using?

14.16.1

### What did you expect to happen?

I expected an error pointing to an invalid value. Only numeric values should be allowed, per config.

If I **DO NOT** make a typo and type: 

`
a {
  font-weight: bold;
}
`

An error is properly raised.

### What actually happened?

No errors

### Proposal to fix the bug

Shouldn't the bug be reported the same way when I input "bold" instead of "bld"?
